automation/src/test/scala/org/broadinstitute/dsde/automation/browser/WebBrowserSpec.scala

```diff
@@ -0,0 +1,112 @@
+package org.broadinstitute.dsde.automation.browser
```

Comment:
Does not contain anything Firecloud-specific and does not depend on anything from the firecloud package